src/components/payment/card/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useStripe, useElements, CardElement } from "@stripe/react-stripe-js";
+import { useTranslation } from 'react-i18next';
+
+import { ProfileContent } from '../../profile/content';
+import { PaymentBottom } from '../bottom';
+import { Button } from '../../ui/Button';
+
+import style from './styles.module.scss';
+
+export const PaymentCard = ({ onChangeStep, currentStep, steps }) => {
+  const { t } = useTranslation();
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const handleSubmit = async event => {
+    event.preventDefault();
+
+    if (!stripe || !elements) {
+      return;
+    }
+    const payload = await stripe.createPaymentMethod({
+      type: "card",
+      card: elements.getElement(CardElement)
+    });
+    console.log("[PaymentMethod]", payload);
```

Comment:
Make at least a TODO to remove that line later

Reply:
Added display for results